manager/controllers/advisories_test.go

```diff
@@ -321,3 +321,25 @@ func TestAdvisoryTagsInMetadata(t *testing.T) {
 	}
 	assert.Equal(t, testMap, output.Meta.Filter)
 }
+
+func TestAdvisoryMetadataSums(t *testing.T) {
+	output := testAdvisories(t, "/")
+	var other, enhancement, bugfix, security int
+	for _, ai := range output.Data {
+		switch ai.Attributes.AdvisoryType {
+		case 1:
+			enhancement++
+		case 2:
+			bugfix++
+		case 3:
+			security++
+		default:
+			other++
+		}
+	}
+	st := output.Meta.SubTotals
+	assert.Equal(t, st["other"], other)
+	assert.Equal(t, st["enhancement"], enhancement)
```

Comment:
Just a last nitpick: assert argument order is `expected`, `found` so in this case it should be
`assert.Equal(t, enhancement, st["enhancement"])`
and similarly for other asserts here.

Reply:
Very good catch, thanks!